hooks/pre_validations/12-check_capsule_tar.rb

```diff
@@ -4,8 +4,12 @@ def error(message)
   kafo.class.exit 101
 end
 
-if param('capsule', 'certs_tar') && (certs_tar = param('capsule', 'certs_tar').value)
-  unless File.file?(certs_tar)
-    error "The certs tar file generated by the server is not present at #{certs_tar}, exiting."
+certs_tar = param('foreman_proxy_certs', 'certs_tar') ||
+  param('foreman_proxy_content', 'certs_tar')
+  
+if certs_tar.value
+  certs_tar.value = File.expand_path(certs_tar.value)
```

Comment:
Do you know if `expand_path` handles `~` and thus also solves that issue we've avoided through docs?

Reply:
Yes, it handles `~` as well